scrc/dataset_creation/judgment_dataset_creator.py

```diff
@@ -49,8 +57,48 @@ def plot_custom(self, df, split_folder, folder):
         self.plot_labels(df, split_folder, label_name='label')
 
 
+def create_multiple_datasets(court_names, exclude_courts):
+    not_created = []
+    created = []
+    for court_string in tqdm(court_names):
+        if court_string not in exclude_courts:
+            print(f"Creating dataset for {court_string}")
+            got_created = judgment_dataset_creator.create_dataset(court_string=court_string, sub_datasets=False, kaggle=False,
+                                                            save_reports=True)
+            if got_created:
+                # rename judgment_prediction folder to {court_string}
+                source = str(ROOT_DIR) + '/data/datasets/judgment_prediction'
+                destination = str(ROOT_DIR) + '/data/datasets/' + court_string
```

Comment:
please use the paths that are available as instance variables already

Reply:
Done 👍